src/lib/utilities/focus-trap.ts

```diff
@@ -0,0 +1,77 @@
+export const getFocusableElements = (node: HTMLElement) =>
+  Array.from(
+    node.querySelectorAll<
+      | HTMLLinkElement
+      | HTMLButtonElement
+      | HTMLTextAreaElement
+      | HTMLInputElement
+      | HTMLDivElement
+      | HTMLSelectElement
+    >('[href], button, textarea, input, div[contentEditable="true"], select'),
+  ).filter(
+    (element) =>
+      !element.hasAttribute('disabled') &&
+      !(element.getAttribute('tabindex') === '-1'),
```

Comment:
Tried adding `[tabindex]:not([tabindex="-1"])` to the query selector argument, but it didn't appear to work based on testing.